minecraft_servers/hybox/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "server_name": "hybox",
+  "nice_name": "Hybox Network",
+  "direct_ip": "hybox.gg",
+  "supported_languages": [
+     "es"
+  ],
+  "social": {
+    "web": "https://hybox.gg",
+    "web_shop": "https://tienda.hybox.gg",
+    "web_support": "https://hybox.gg",
+    "twitter": "HyboxNetwork",
+    "discord": "https://discord.hybox.gg"
```

Comment:
As custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url